doc/readme.md

```diff
@@ -405,9 +401,9 @@ based on the following criteria (listed in order of precedence):
 
         cordova plugin rm camera --nosave
 
-- Remove the plugin from the project and `npm uninstall` it. Removes it from the `node_modules` directory:
+- Do not remove the plugin from the project and `npm uninstall` it. Do not remove it from the `node_modules` directory:
```

Comment:
can we update this to say "Remove the plugin from the project.  Do not remove it from the `node_modules` directory and don't run `npm uninstall` under the hood: